core-bundle/src/Resources/contao/dca/tl_undo.php

```diff
@@ -113,6 +113,10 @@
 		(
 			'search'                  => true,
 			'sql'                     => "mediumblob NULL"
+		),
+		'preview' => array
+		(
+			'sql'                     => "mediumblob NULL"
```

Comment:
```suggestion
			'eval'                    => array('doNotShow'=>true),
			'sql'                     => "mediumblob NULL"
```

This will hide the field in the "show" view.